### ISSUE TITLE
Audio TextTrack not visible

### ISSUE THREAD

**Issue by buritica** [videojs/video.js#2854]
Hey all,

I'm trying to add a `<track>` to an audio element but I can't get it to display. Same configuration works perfectly on a video element as you can see in this [JSBin](http://jsbin.com/piyexi/)

I'm not sure if I'm doing something wrong, there is no support for this yet or there is an issue.

Thanks!

**Comment by buritica** [comment]
Hi, any ideas on where I could find any information about this? 

Is there `<audio>` support for trackas the same way as `<video>`? I can take a look at the source if you confirm it's a bug, or implementing it if it isn't but I'd need some guidance on your part.

**Comment by gkatsev** [comment]
I'm able to see the text track for audio locally.
The interesting thing here, though, is that the specification for the media elements specifically say that the difference between audio and video is that video has area to display things such as text tracks whereas audio doesn't.
So, you could probably just use a `<video>` element just as well, at least as a workaround.
We should probably decide whether we want to follow the spec on this one or not.

**Comment by buritica** [comment]
Is locally with a new version? Cause I can't get it locally or on that jsbin and it's driving me a little nuts :D.

Is your suggestion to use a video element, but with an audio source? Haven't tried that, if it works then I can probably just hide some of the stuff so I can show the subtitles.

Here's my reasoning and problem I'm trying to solve:

I'm trying to play videos, and audios from a randomly generated playlist so I have 2 media playing at all times and that's why I need the track timecode of the sutbitles shown on the video to match the audio track. If I can get the `<audio>` ones to show, I can superimpose them over the video. If I can't, but having another `<video>` element as you suggest works, then I could try to only show the track on top of the other video and problem solved.

Any alternative approaches you can suggest are more than welcome, really enjoying using videojs thanks for all the hard work on this!

**Comment by gkatsev** [comment]
Yeah, locally is with a newer version. I think if using a video element for this works, you should go with that, especially considering the spec's recommendation on the subject.
Closing since we probably won't change anything in videojs, but we can continue the discussion if necessary.

**Comment by buritica** [comment]
Thanks,

I agree with closing.

What version were you using?

**Comment by gkatsev** [comment]
Heh, I don't even remember. Whatever was in `master` on december 5th :)

**Comment by OwenEdwards** [comment]
On another thread, we confirmed that a text track added to an `<audio>` element is not displayed (I checked with a member of the W3C), so this is the expected behavior for video.js. It was recommended to just use a `<video>` element (perhaps with a poster), and the same media sources, to make the track(s) work as expected. Video.js could throw a console warning message if a user tries to enable a track when there's nowhere to display it? Other than that, the only change we would want to make is to add this to some, ahem, documentation somewhere :-)
